File: features/rolemanagement/src/main/java/com/nivasafinance/features/rolemanagement/permissiongroup/service/PermissionGroupReadService.java
package com.nivasafinance.features.rolemanagement.permissiongroup.service;

import com.nivasafinance.features.rolemanagement.permissiongroup.dto.PermissionGroupResponse;

import java.util.List;

public interface PermissionGroupReadService {
    List<PermissionGroupResponse> getPermissionGroupsbyRoles(List<String> roleName);
}

File: features/rolemanagement/src/main/java/com/nivasafinance/features/rolemanagement/permissiongroup/service/PermissionGroupReadService.kt
package com.nivasafinance.features.rolemanagement.permissiongroup.service

import com.nivasafinance.features.rolemanagement.permissiongroup.dto.PermissionGroupResponse

interface PermissionGroupService {
    fun getPermissionGroupsbyRoles(roleName: List<String>): List<PermissionGroupResponse>
}
